src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/regelsett/lov2021/barnehageplass/Graderingsforskjell.kt

```diff
@@ -9,3 +13,35 @@ enum class Graderingsforskjell {
     LIK,
     INGEN_UTBETALING_GRUNNET_FØRSTE_PERIODE_TIL_ØKNING,
 }
+
+fun finnGraderingsforskjellMellomDenneOgForrigePeriode(
+    vilkårResultatForrigePeriode: VilkårResultat?,
+    vilkårResultatDennePerioden: VilkårResultat?,
+    erFørstePeriode: Boolean,
+): Graderingsforskjell {
+    val graderingForrigePeriode =
+        when (vilkårResultatForrigePeriode == null) {
+            false -> hentProsentForAntallTimer(vilkårResultatForrigePeriode.antallTimer)
+            true -> BigDecimal.ZERO
+        }
+
+    val graderingDennePerioden =
+        when (vilkårResultatDennePerioden == null) {
+            false -> hentProsentForAntallTimer(vilkårResultatDennePerioden.antallTimer)
+            true -> BigDecimal.ZERO
```

Comment:
When's med 2 brancher + som sjekker på false true synes jeg bør være if's for lesbarhet.

Reply:
ref https://kotlinlang.org/docs/coding-conventions.html#if-versus-when

Reply:
Enig! Veldig god link også! 🙌